config/database.yml

```diff
@@ -24,9 +25,5 @@ test:
   database: jobstatus_test
 
 production:
-  Host:
-  port:
```

Comment:
In production we'll want to be able to access the database as well.  The default config would likely be a better place.